migrations/README.rst

```diff
@@ -242,4 +253,44 @@ You should upgrade your DB and then run the revision command:
 .. code-block:: bash
 
     alembic upgrade head
-    alembic revision -m "your message" --autogenerate
\ No newline at end of file
+    alembic revision -m "your message" --autogenerate
+
+------------
+
+**FOR DEVELOPERS ONLY**
+
+.. code-block:: bash
+
+    Database tables are not found! (Hint: Did you initialise the DataStore?)
+
+If you migrated your **SQLite** database and started to see this message, it might mean that your migration script it wrong.
```

Comment:
Very minor typo here - should be "is" rather than "it"

Reply:
Thanks, @robintw! Typo fixed: https://github.com/debrief/pepys-import/pull/427/commits/98e6d4a4e9beb19f098d78951309cb36a1de3c04